apps/desktop-wallet/src/features/addressDeletion/DeleteAddressesModal.tsx

```diff
@@ -123,9 +126,13 @@ const DeleteAddressesModal = memo(({ id }: ModalBaseProp) => {
           isLoading={isLoadingSortedAddresses}
           modalId={id}
         />
-      </ModalFooterButtons>
+      </ModalFooterButtonsStyled>
     </CenteredModal>
   )
 })
 
 export default DeleteAddressesModal
+
+const ModalFooterButtonsStyled = styled(ModalFooterButtons)`
+  margin: var(--spacing-4) var(--spacing-6);
```

Comment:
I 100% agree

Reply:
I didn't want to spend more time on it now, knowing that a redesign is coming.

Reply:
Could you just adjust to :

`var(--spacing-2) var(--spacing-6)` 

Then we can merge! LGTM :)

Reply:
And.. I'm sorry, but could you also quickly change the address list so that everything is aligned and without shadow? 🙏 

<img width="631" alt="image" src="https://github.com/user-attachments/assets/4a72a25b-d2fc-4323-9679-f5f1f99a7800">

Reply:
Done and pushed 🫸 ✅